.github/workflows/deployment.yml

```
@@ -0,0 +1,70 @@
+name: Deploy FrontEnd
+
+on:
+  push:
+    branches: [ stage, main, release/** ]
```

```suggestion
  push:
    branches: [main, stage]
    tags:
      - "v*"
```
Talvez seja melhor validar por tag em vez de uma branch chamada release